theme/less/_navigation.less

```diff
@@ -1,26 +1,82 @@
-nav
+header nav
 {
-  ul
+  border-top: 1px solid #E9E9E9;
+  background: #FFFFFF;
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+  margin: 25px 0;
+}
+
+header nav ul
+{
+  display: flex;
+  margin: 0;
+  padding: 0 30px;
+  align-items: center;
+  list-style: none;
+  height: 78px;
+}
+
+header nav ul a
+{
+  font-weight: 500;
+  font-size: 24px;
+  text-decoration: none;
+  color: #626262;
+  padding: 10px 45px;
+}
+
+#hamburger
+{
+  background-color: transparent;
+  cursor: pointer;
+  padding: 0.5rem;
+  border: 0;
+}
+
+#hamburger:before
+{
+ content: "";
+ display: block;
+ background-image: url(/images/hamburger.svg);
+ height: 2rem;
+ width: 2rem;
+}
+
+#hamburger-menu
+{
+  text-align: center; 
+}
+
+#hamburger-menu ul
```

Comment:
Meeting notes:
- Keep bottom border shadow, to make menu jump more expected.

Reply:
Mean to keep the border bottom consistent on mobile devices, similar to CMintS.

Reply:
Done.